RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -151,9 +150,51 @@ void createOptionalOutputBranches() {
   ana.tx->createBranch<int>("pT3_occupancies");
   ana.tx->createBranch<int>("pT5_occupancies");
 
+  // T5 DNN branches
+  createT5DNNBranches();
+
 #endif
 }
 
+//________________________________________________________________________________________________________________________________
+void createT5DNNBranches() {
+  // T3 branches for T5 DNN
+  ana.tx->createBranch<std::vector<int>>("t5_t3_idx0");
+  ana.tx->createBranch<std::vector<int>>("t5_t3_idx1");
+  ana.tx->createBranch<std::vector<int>>("t5_tc_idx");
+  ana.tx->createBranch<std::vector<int>>("t5_partOfTC");
+  ana.tx->createBranch<std::vector<float>>("t3_pt");
+  ana.tx->createBranch<std::vector<float>>("t3_eta");
+  ana.tx->createBranch<std::vector<float>>("t3_phi");
+  ana.tx->createBranch<std::vector<float>>("t3_0_r");
+  ana.tx->createBranch<std::vector<float>>("t3_0_x");
```

Comment:
these should be named better to preserve relationship with `t5s`, to avoid confusion with possible plain `t3` branches. Could be just a `t5_` prefix everywhere